src/graph_compute/compute.cpp

```diff
@@ -101,6 +127,9 @@ WHEN_TEST({
     cout << "pthread workers: " << NUMA_WORKERS << '\n';
   #elif PARALLEL
     cout << "Cilk workers: " << __cilkrts_get_nworkers() << '\n';
+    #ifndef NUMA_WORKERS
```

Comment:
I really don't like this. NUMA_WORKERS is overridden here, but only if VERBOSE and PARALLEL are set, and in the middle of a printing statement. I firmly believe in all #define statements existing before code.